### PR TITLE
[Playwright test fix] LPD-31952

### PR DESCRIPTION
**Pull request by p-albuquerque**
Related issue: https://issues.liferay.com/browse/LPD-31952

### CODE REVIEW

**Comment by p-albuquerque** [comment]
ci:test:playwright

**Comment by marcelabc** [comment]
Hey @danvb112 
Could you review this PR, please? :smile:

**Comment by danvb112** [comment]
Sure @marcelabc! I'll start the review now!

File: modules/test/playwright/tests/object-web/objectFields.spec.ts [hunk 1]
```diff
@@ -93,14 +93,17 @@ test.describe('Manage object fields through Model Builder', () => {
...
-			Array(22)
-				.fill(null)
-				.map(() =>
-					apiHelpers.listTypeAdmin.postRandomListTypeDefinition()
-				)
-		);
+				
+		let listTypeDefinitions = (await apiHelpers.listTypeAdmin.getListTypeDefinitions()).items;
+
+		listTypeDefinitions = listTypeDefinitions.concat(
```
Comment (danvb112) [inline modules/test/playwright/tests/object-web/objectFields.spec.ts:99]: Hey @p-albuquerque, I think it's better to create a new const with a different name, something like `allListTypeDefinitions`

File: modules/test/playwright/tests/object-web/objectFields.spec.ts [hunk 2]
```diff
@@ -93,14 +93,17 @@ test.describe('Manage object fields through Model Builder', () => {
...
+			await Promise.all(
+				Array(22)
+					.fill(null)
+					.map(() =>
+						apiHelpers.listTypeAdmin.postRandomListTypeDefinition()
+					)
+			));
```
Comment (danvb112) [inline modules/test/playwright/tests/object-web/objectFields.spec.ts:106]: Hey @p-albuquerque, I know this code was here before, but we should be using an async `map` to make sense of using `Promise.all`, so we need to rewrite that map to make sure this async code is waiting correctly:

```js
await Promise.all(
  Array(22)
	  .fill(null)
	  .map(async () =>
		  await apiHelpers.listTypeAdmin.postRandomListTypeDefinition()
	)
));
```

**Review by danvb112 — CHANGES_REQUESTED** [review]
Hey @p-albuquerque, great job! Just two suggestions :smile:

**Comment by p-albuquerque** [comment]
ci:test:sf

**Comment by p-albuquerque** [comment]
ci:test:playwright

File: modules/test/playwright/tests/object-web/objectFields.spec.ts [hunk 3]
```diff
@@ -94,15 +94,24 @@ test.describe('Manage object fields through Model Builder', () => {
...
-			Array(22)
-				.fill(null)
-				.map(() =>
-					apiHelpers.listTypeAdmin.postRandomListTypeDefinition()
-				)
+		let allListTypeDefinitions = (
+			await apiHelpers.listTypeAdmin.getListTypeDefinitions()
+		).items;
+
+		allListTypeDefinitions = allListTypeDefinitions.concat(
```
Comment (danvb112) [inline modules/test/playwright/tests/object-web/objectFields.spec.ts:101]: Hey @p-albuquerque, could you create two const `existingListTypeDefinitions` and `allListTypeDefinitions` to make the code more readable?

```ts
const existingListTypeDefinitions = (
	await apiHelpers.listTypeAdmin.getListTypeDefinitions()
).items;

const allListTypeDefinitions = existingListTypeDefinitions.concat(

...

```

**Review by danvb112 — CHANGES_REQUESTED** [review]
Hey @p-albuquerque, just one more thing

**Comment by p-albuquerque** [comment]
ci:test:sf

**Comment by p-albuquerque** [comment]
ci:test:sf

**Comment by p-albuquerque** [comment]
ci:test:playwright

**Review by danvb112 — APPROVED** [review]
LGTM! :tada:

**Comment by p-albuquerque** [comment]
ci:forward

**Pull request closed without merging**